linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/ansible/shared.yml

```diff
@@ -5,21 +5,27 @@
 # disruption = low
 {{{ ansible_instantiate_variables("rsyslog_remote_loghost_address") }}}
 
-- name: "Get omfwd configuration directive"
-  shell: sed -e '/^action\s*(\s*type\s*=\s*"omfwd"/,/)/!d' /etc/rsyslog.conf /etc/rsyslog.d/*.conf || true
-  register: include_omfwd_config_output
+- name: "{{{ rule_title }}}: Find all files matching /etc/rsyslog.d/*.conf"
+  find:
+    paths: "/etc/rsyslog.d/"
+    pattern: "*.conf"
+  register: rsyslog_includes
 
-- name: "Get include files directives"
-  shell: >
-    set -o pipefail
-    echo \"{{ include_omfwd_config_output.stdout }}\"|grep  'StreamDriver=\"gtls\"'
-  register: include_omfwd_gtls_config_output
-  when: (include_omfwd_config_output.stdout_lines| length > 0)
+- name: "{{{ rule_title }}}: Retrieve list of Rsyslog config files"
+  set_fact:
```

Comment:
Could you use the complete name for the module, please?
In this case: `ansible.builtin.set_fact`